app/src/main/java/vn/mbm/phimp/me/leafpic/data/providers/MediaStoreProvider.java

```diff
@@ -130,6 +130,7 @@ private static int getAlbumCount(Context context, long id) {
 										   + MediaStore.Files.FileColumns.MEDIA_TYPE + "=? ) and " + MediaStore.Files.FileColumns.PARENT + "=?";
 
 		String[] selectionArgs = new String[] {
+						String.valueOf(MediaStore.Files.FileColumns.MEDIA_TYPE),
```

Comment:
@anantprsd5 The issue got fixed but can you explain what exactly have you done here?